process-migration-service/src/test/filtered-resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  version="2.1"
+  xmlns="http://xmlns.jcp.org/xml/ns/persistence"
+  xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
+  <persistence-unit name="org.kie.test.persistence" transaction-type="JTA">
+    <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+    <jta-data-source>jdbc/testDS1</jta-data-source>
+    <class>org.kie.processmigration.model.Plan</class>
+    <class>org.kie.processmigration.model.Migration</class>
+    <class>org.kie.processmigration.model.MigrationReport</class>
+    <properties>
+      <property name="hibernate.dialect" value="${maven.hibernate.dialect}"/>
+      <property name="hibernate.default_schema" value="${maven.jdbc.schema}"/>
+
+      <property name="hibernate.max_fetch_depth" value="3"/>
+      <property name="hibernate.hbm2ddl.auto" value="create-drop"/>
```

Comment:
Hmm, what about leaving just `create` here? It might be useful to have the DB content available after tests.

Reply:
done